.pre-commit-config.yaml

```diff
@@ -93,7 +93,18 @@ repos:
         pass_filenames: false
         types: [python]
         stages: [manual]
-        additional_dependencies: ['pyright@1.1.248']
+        additional_dependencies: ['pyright@1.1.252']
+-   repo: local
+    hooks:
+    -   id: pyright_reportGeneralTypeIssues
+        name: pyright reportGeneralTypeIssues
+        entry:  pyright --skipunannotated -p pyright_reportGeneralTypeIssues.json
```

Comment:
Is it preferable to combine the two pyright calls in one hook? Something like `pyright && pyright --skipunannotated -p pyright_reportGeneralTypeIssues.json` @MarcoGorelli

Reply:
i don't know (no preference from me)

Reply:
Okay. Similar to the flake8 dependencies, I share now the (one) pyright dependency: need to change only one line to update pyright in the future.